src/cli.cr

```diff
@@ -40,17 +40,25 @@ module Wsman
         sub "setup" do
           desc "Generate site configurations for the given site."
           usage "setup [options] <sitename>"
+          option "-e ENVFILE", "--envfile ENVFILE", type: String, required: false, desc: "Extra environment file for docker container."
           run do |_opts, args|
```

Comment:
The `_` prefix on variables are used to indicate a mandatory, but unused variable. Since you are using it in this block, please rename it to `opts`.